src/rgw/driver/rados/rgw_bucket.cc

```diff
@@ -783,6 +1099,48 @@ int RGWBucketAdminOp::chown(rgw::sal::Driver* driver, RGWBucketAdminOpState& op_
 
 }
 
+int RGWBucketAdminOp::check_index_olh(rgw::sal::Driver* driver, RGWBucketAdminOpState& op_state,
+                  RGWFormatterFlusher& flusher, const DoutPrefixProvider *dpp)
+{
+  RGWBucket bucket;
+  int ret = bucket.init(driver, op_state, null_yield, dpp);
+  if (ret < 0) {
+    ldpp_dout(dpp, -1) << "bucket.init(): " << ret << dendl;
+    return ret;
+  }
+  flusher.start(0);
+  uint64_t total_count;
+  ret = bucket.check_index_olh(dpp, op_state, flusher, &total_count);
```

Comment:
`bucket.check_index_olh()` is what logs the value of `total_count`, so it doesn't look like it needs to be an output parameter. same for `check_index_unlinked()`